src/docx_cr_converter/macros.cr

```diff
@@ -0,0 +1,27 @@
+macro heading(header, document)
+  {% begin %}
+    if node.as(XML::Node).xpath_nodes("w:pStyle/@w:val").to_s.includes?({{header}})
```

Comment:
Hi @onliniak alex could you adjust the indentation to two spaces? This can help in reading!

```crystal
if node.as(XML::Node).xpath_nodes("w:pStyle/@w:val").to_s.includes?({{header}})
  main_node = node.parent.as(XML::Node).xpath_nodes("w:r/w:t")
  word = main_node.to_s[5 .. -7]
  @document += {{document}}
end
```

Reply:
Corrected